src/components/layers/legend/LegendItem.js

```diff
@@ -23,9 +25,11 @@ const LegendItem = ({
     };
 
     if (radius) {
-        symbol.width = radius * 2 + 'px';
-        symbol.height = radius * 2 + 'px';
-        symbol.borderRadius = radius ? '50%' : '0';
+        const r = (radius < maxRadius ? radius : maxRadius) * 2;
```

Comment:
Should this just be `Math.min(radius, maxRadius) * 2`?

Reply:
True, fixed!